temoa/model_checking/unit_checking/screener.py

```
@@ -0,0 +1,321 @@
+    report_entries = []
+
+    for db_path in db_paths:
+        if not db_path.is_file():
+            raise FileNotFoundError(f'Database file not found: {db_path}')
+        initialization_msg = f'\n========  Units Check on DB: {db_path}:  Started ========\n\n'
+        report_entries.append(initialization_msg)
+        logger.info('Starting Units Check on DB: %s', db_path)
+
+        with sqlite3.connect(db_path) as conn:
+            # test 1: DB version
+            db_version_ok, major_version, minor_version = _check_db_version(conn, report_entries)
```

Ruff (0.14.8)

Unpacked variable `major_version` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)